packages/sourcecred/src/core/ledger/ledger.js

```diff
@@ -61,12 +63,21 @@ type AllocationReceipt = {|
   +credTimestampMs: TimestampMs,
 |};
 
+/**
+ * The state of the Ledger's accounting configuration.
+ */
+export type AccountingStatus = {|
+  +enabled: boolean,
+  +trackDistributions: boolean,
+  +currency: ?Currency,
+|};
 /**
  * Every Identity in the ledger has an Account.
  */
 type MutableAccount = {|
   identity: Identity,
-  // The current Grain balance of this account
+  // The current Grain balance of this account. All balances are zero when
```

Comment:
Reminder that we should start thinking about using and substituting /** */ for the docs, including attributes I believe.